pkg/roll/update_test.go

```diff
@@ -683,8 +684,88 @@ func TestShouldRollMidwayHealthyMigrationFromZeroWhenNewSatisfies(t *testing.T)
 	upd.DesiredReplicas = 3
 	upd.MinimumReplicas = 2
 
-	remove, add, err := upd.shouldRollAfterDelay(rc_fields.RC{ID: upd.NewRC, Manifest: manifest})
+	remove, add, err := upd.shouldRollAfterDelay(manifest.ID())
 	Assert(t).IsNil(err, "expected no error determining nodes to roll")
 	Assert(t).AreEqual(remove, 0, "expected to remove no nodes")
 	Assert(t).AreEqual(add, 1, "expected to add one node")
 }
+
+func panicOnError(desc string, errs <-chan error) {
+	err, ok := <-errs
+	if ok {
+		panic("Error received on " + desc + ":" + err.Error())
```

Comment:
it maybe would; I'd just need to pass the `testing.T`. Will need to figure out how to test since I don't have anything that exercises it right now